test/scenarios-bootc/periodics/el94-crel@published-images-standard2.sh

```diff
@@ -46,7 +43,10 @@ scenario_run_tests() {
         # TODO: While 4.19-ec is not available, it needs to exit without an error.
         exit 0
     fi
+    # Until 4.19 EC starts including correct default config,
```

Comment:
I presume this is backport? This is 4.18 branch, so we're waiting until next Z-stream release.
If we have idea when that will happen, we could make that exclude time dependent - active until some date?

Reply:
It is a backport. I will create blocked bugs for reactivating the tests both in main and 4.18 branches.
Will post issue references here before resolving this thread.

Reply:
See [USHIFT-5526](https://issues.redhat.com/browse/USHIFT-5526) and  [USHIFT-5527](https://issues.redhat.com/browse/USHIFT-5527).

Would you like me to fix the comment in 4.18?